lib/impl/CosmosStateStore.js

```diff
@@ -112,9 +113,16 @@ class CosmosStateStore extends StateStore {
       }))
     }
 
+    const inMemoryInstance = CosmosStateStore.inMemoryInstance[credentials.partitionKey]
```

Comment:
I believe its an ISO string.

Reply:
`state.init` will call `aio-lib-core-tvm` which already takes care of doing an expiration check. If the token is expired it will be generated otherwise the cached token is returned to state lib. So the comparison here is to check if a new token was issued.
Later we could iterate on this and do an expiration check on every state request to cover cases where the token expires in-between two init calls (less likely).